Gemfile.lock

```diff
@@ -119,155 +124,162 @@ GEM
-    parser (3.1.2.0)
+    parser (3.1.3.0)
       ast (~> 2.4.1)
-    pry (0.13.1)
+    pry (0.14.1)
       coderay (~> 1.1)
       method_source (~> 1.0)
-    pry-byebug (3.9.0)
+    pry-byebug (3.10.1)
       byebug (~> 11.0)
-      pry (~> 0.13.0)
-    public_suffix (4.0.7)
-    puma (5.6.4)
+      pry (>= 0.13, < 0.15)
+    public_suffix (5.0.0)
+    puma (5.6.5)
       nio4r (~> 2.0)
-    racc (1.6.0)
+    racc (1.6.1)
     rack (2.2.4)
     rainbow (3.1.1)
     rake (13.0.6)
-    redis (4.7.1)
-    regexp_parser (2.5.0)
+    redis (4.8.0)
+    regexp_parser (2.6.1)
+    reline (0.3.1)
+      io-console (~> 0.5)
     rest-client (2.1.0)
       http-accept (>= 1.7.0, < 2.0)
       http-cookie (>= 1.0.2, < 2.0)
       mime-types (>= 1.16, < 4.0)
       netrc (~> 0.8)
     rexml (3.2.5)
-    rspec (3.11.0)
-      rspec-core (~> 3.11.0)
-      rspec-expectations (~> 3.11.0)
-      rspec-mocks (~> 3.11.0)
-    rspec-core (3.11.0)
-      rspec-support (~> 3.11.0)
-    rspec-expectations (3.11.0)
+    rspec (3.12.0)
+      rspec-core (~> 3.12.0)
+      rspec-expectations (~> 3.12.0)
+      rspec-mocks (~> 3.12.0)
+    rspec-core (3.12.0)
+      rspec-support (~> 3.12.0)
+    rspec-expectations (3.12.0)
       diff-lcs (>= 1.2.0, < 2.0)
-      rspec-support (~> 3.11.0)
-    rspec-mocks (3.11.1)
+      rspec-support (~> 3.12.0)
+    rspec-mocks (3.12.0)
       diff-lcs (>= 1.2.0, < 2.0)
-      rspec-support (~> 3.11.0)
-    rspec-support (3.11.0)
-    rubocop (1.32.0)
+      rspec-support (~> 3.12.0)
+    rspec-support (3.12.0)
+    rubocop (1.39.0)
       json (~> 2.3)
       parallel (~> 1.10)
-      parser (>= 3.1.0.0)
+      parser (>= 3.1.2.1)
       rainbow (>= 2.2.2, < 4.0)
       regexp_parser (>= 1.8, < 3.0)
       rexml (>= 3.2.5, < 4.0)
-      rubocop-ast (>= 1.19.1, < 2.0)
+      rubocop-ast (>= 1.23.0, < 2.0)
       ruby-progressbar (~> 1.7)
       unicode-display_width (>= 1.4.0, < 3.0)
-    rubocop-ast (1.19.1)
+    rubocop-ast (1.24.0)
       parser (>= 3.1.1.0)
-    rubocop-rspec (2.12.1)
-      rubocop (~> 1.31)
+    rubocop-rspec (2.15.0)
+      rubocop (~> 1.33)
     ruby-progressbar (1.11.0)
     ruby2_keywords (0.0.5)
     sequel (5.42.0)
-    sidekiq (6.4.2)
-      connection_pool (>= 2.2.2)
+    sidekiq (6.5.8)
+      connection_pool (>= 2.2.5, < 3)
       rack (~> 2.0)
-      redis (>= 4.2.0)
-    sqlite3 (1.4.4)
-    thor (1.1.0)
-    tilt (2.0.10)
-    transproc (1.1.1)
+      redis (>= 4.5.0, < 5)
+    sqlite3 (1.5.4-x86_64-darwin)
+    sqlite3 (1.5.4-x86_64-linux)
+    thor (1.2.1)
+    tilt (2.0.11)
     tzinfo (2.0.5)
       concurrent-ruby (~> 1.0)
     unf (0.1.4)
       unf_ext
     unf_ext (0.0.8.2)
-    unicode-display_width (2.2.0)
-    url_mount (0.2.1)
-      rack
-    webmock (3.14.0)
+    unicode-display_width (2.3.0)
+    webmock (3.18.1)
       addressable (>= 2.8.0)
       crack (>= 0.3.2)
       hashdiff (>= 0.4.0, < 2.0.0)
-    zeitwerk (2.6.0)
+    zeitwerk (2.6.6)
 
 PLATFORMS
-  aarch64-linux
```

Comment:
When I run `bundle install`, `ruby` is added here.

Reply:
Added the ruby platform back in https://github.com/projecttacoma/deqm-test-kit/pull/42/commits/45db51985c701615d7eb2c0c42c32b1cc5e38359

My thought process here: Based on the bundler docs, `PLATFORMS` represents "The ruby platform the Gemfile.lock was generated against. If any dependencies in the Gemfile specify a platform they they will only be included in that Gemfile.lock when the lock file is generated on that platform (e.g. through an install)." So, when I did an install it removed the `ruby` platform, thus removing the dependencies that you were getting when you did an install.

To bring the ruby platform back, I ran `bundle lock --add-platform ruby`. Pretty sure this updated the dependencies automatically, or maybe I had to run `bundle update` after - either way, it added those dependencies back in.

Based on this (probably not super reliable but idk)[ source](https://www.moncefbelyamani.com/understanding-the-gemfile-lock-file/ ), “Before, when projects were bundled with Bundler version 2.1.x, the platforms section in Gemfile.lock would automatically include the ruby platform, along with the current operating system. This allowed Ruby apps to be deployed on Linux servers. But since version 2.2, the ruby platform is no longer included by default.”

So, this might just be a result of the bundler version that I'm using. I did not find a ton of information about the ruby platform as a whole - not enough to determine if we actually need to include it for this project - but the [inferno-core template's lock file ](https://github.com/inferno-framework/inferno-template/blob/main/Gemfile.lock) and the [us-core-test-kit's lock file](https://github.com/inferno-framework/us-core-test-kit/blob/main/Gemfile.lock) (another test kit that uses the template) both include the ruby platform, so I'm going to use those as precedent to say we should just keep the ruby platform in for now because it doesn't hurt